Trying to run tests gives an error

When running `npm run test` I get this:

```
> umzug@1.9.1 test /home/bjn/src/umzug
> gulp

module.js:341
    throw err;
    ^

Error: Cannot find module 'jshint/src/cli'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/bjn/src/umzug/node_modules/gulp-jshint/src/extract.js:1:79)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17
```

--- a/package.json
+++ b/package.json
@@ -17,6 +17,7 @@
     "gulp-jshint": "^2.0.0",
     "gulp-jshint-instafail": "^1.0.0",
     "gulp-mocha": "^2.0.0",
+    "jshint": "^2.9.1",
     "jshint-stylish": "^2.0.0",
     "mocha": "^2.0.1",
     "sequelize": "^3.13.0",

